packages/cursorless-org-docs/src/docs/user/languages/c.mdx

```diff
@@ -1,4 +1,4 @@
-import Language from "./Language";
+import { Language } from "./components/Language";
```

Comment:
We're getting more components so I move them into a component's folder. I also prefer named imports.

Reply:
agreed on all counts